### DISCUSSION TITLE
Welcome to the `great_tables` Discussions!

### DISCUSSION THREAD

**Discussion by rich-iannone** [posit-dev/great-tables#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ankitAPL** [comment]
Hey , I was waiting for this in python for such a long time . Had used 'gt' in R and loved it .
BTW - How can I save output table into png or  pdf  ? 

Thanks
Ankit

**Comment by Breza** [comment]
Very excited to explore this package. Are there any caveats to using it in a Jupyter notebook?

**Comment by ClimberMel** [comment]
I've played with a couple of examples.  Is it expected that this be run in a jupytr notebook?  If I add a print statement, the output is the basic dataframe and what looks similar to HTML.
